From b0edfe1e5bb1a705fad539e5ab1c299d38dcf9d4 Mon Sep 17 00:00:00 2001
From: Andrea Mazzoleni <amadvance@gmail.com>
Date: Tue, 8 May 2018 18:55:39 +0200
Subject: [PATCH 1/3] Use acinclude instead of aclocal

aclocal.m4 is generated automatically, and we don't have to lose the content.
---
 acinclude.m4 | 20 ++++++++++++++++++++
 aclocal.m4   | 29 ++++++++++++-----------------
 2 files changed, 32 insertions(+), 17 deletions(-)
 create mode 100644 acinclude.m4

diff --git a/acinclude.m4 b/acinclude.m4
new file mode 100644
index 00000000..ceed823a
--- /dev/null
+++ b/acinclude.m4
@@ -0,0 +1,20 @@
+dnl @synopsis AC_CHECK_CC_OPT(flag, ifyes, ifno)
+dnl 
+dnl Shows a message as like "checking wether gcc accepts flag ... no"
+dnl and executess ifyes or ifno.
+
+AC_DEFUN([AC_CHECK_CC_OPT],
+[
+AC_MSG_CHECKING([whether ${CC-cc} accepts $1])
+echo 'void f(){}' > conftest.c
+if test -z "`${CC-cc} -c $1 conftest.c 2>&1`"; then
+  AC_MSG_RESULT([yes])
+  $2
+else
+  AC_MSG_RESULT([no])
+  $3
+fi
+rm -f conftest*
+])
+
+
diff --git a/aclocal.m4 b/aclocal.m4
index d6e7b527..56e944be 100644
--- a/aclocal.m4
+++ b/aclocal.m4
@@ -1,20 +1,15 @@
-dnl @synopsis AC_CHECK_CC_OPT(flag, ifyes, ifno)
-dnl 
-dnl Shows a message as like "checking wether gcc accepts flag ... no"
-dnl and executess ifyes or ifno.
+# generated automatically by aclocal 1.15 -*- Autoconf -*-
 
-AC_DEFUN(AC_CHECK_CC_OPT,
-[
-AC_MSG_CHECKING([whether ${CC-cc} accepts $1])
-echo 'void f(){}' > conftest.c
-if test -z "`${CC-cc} -c $1 conftest.c 2>&1`"; then
-  AC_MSG_RESULT([yes])
-  $2
-else
-  AC_MSG_RESULT([no])
-  $3
-fi
-rm -f conftest*
-])
+# Copyright (C) 1996-2014 Free Software Foundation, Inc.
 
+# This file is free software; the Free Software Foundation
+# gives unlimited permission to copy and/or distribute it,
+# with or without modifications, as long as this notice is preserved.
 
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY, to the extent permitted by law; without
+# even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.
+
+m4_ifndef([AC_CONFIG_MACRO_DIRS], [m4_defun([_AM_CONFIG_MACRO_DIRS], [])m4_defun([AC_CONFIG_MACRO_DIRS], [_AM_CONFIG_MACRO_DIRS($@)])])
+m4_include([acinclude.m4])
-- 
2.17.0

